doc/09.-Network-shares.md

```diff
@@ -12,6 +12,8 @@ The mount process for these mounts is triggered at the moment a client logs in.
 
 All protocols supported by the [mount command](https://manpages.ubuntu.com/manpages/jammy/en/man8/mount.8.html) should work out of the box. However, the only tested ones are `smb`, `ftp` and `nfs`.
 
+The backends for the protocols `smb` and `nfs` are automatically enabled when installing the adsys package. In order to enable the backend for `ftp` mounts, the user must install the recommended `curlftpfs` package.
```

Comment:
Maybe mentions that this is valid on Ubuntu? (in case some other distros carries adsys one day)